frontend/pulse3d/src/components/uploads/WaveformGraph.js

```diff
@@ -444,8 +495,12 @@ export default function WaveformGraph({
       .attr("transform", (d) => {
         return "translate(" + x(dataToGraph[d][0]) + "," + (y(dataToGraph[d][1]) - 7) + ") rotate(180)";
       })
-      .style("fill", "orange")
-      .attr("stroke", "orange")
+      .style("fill", (d) => {
```

Comment:
You can change the color of the triangles while dragging in the function on line 429 so that they stay red